operator/internal/controller/pulumi/stack_controller.go

```diff
@@ -1331,3 +1361,14 @@ func patchObject[T any, V any](base T, patch V) (*T, error) {
 
 	return &result, nil
 }
+
+func exactlyOneOf(these ...bool) bool {
```

Comment:
We have this in utils.go already right?

Reply:
Jinxed with #708, but no need to keep that file around for a function that's only used in one place.